src/main/kotlin/com/ktor/solution/Application.kt

```diff
@@ -0,0 +1,202 @@
+package com.ktor.solution
+
+import com.ktor.solution.Config.API_BASE_URL
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.client.plugins.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.client.HttpClient
+import io.ktor.client.call.*
+
+import io.ktor.server.request.*
+import io.ktor.http.*
+
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation as ClientContentNegotiation
+import io.ktor.server.plugins.contentnegotiation.ContentNegotiation as ServerContentNegotiation
+import io.ktor.serialization.kotlinx.json.*
+import kotlinx.serialization.json.Json
+
+import io.ktor.client.plugins.defaultRequest
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Post(val userId: Int, val id: Int, val title: String, val body: String)
+
+@Serializable
+data class Comment(val postId: Int, val id: Int, val name: String, val email: String, val body: String)
+
+object Config {
+    const val API_BASE_URL = "https://jsonplaceholder.typicode.com"
+}
+
+fun main() {
```

Comment:
Please split this logic to at least 2-3 classes

Reply:
done